services/services.md

```diff
@@ -29,6 +29,7 @@ Generic services are each representing an aspect or functionality of a device. T
 - [Scene control](/services/generic/scene_control.md)
 - [Siren control](/services/generic/siren_control.md)
 - [Media player](/services/generic/media_player.md)
+- [Sound switch](/services/generic/sound_switch.md)
```

Comment:
Add it also to _sidebar.md.

Reply:
@PaulinaDrozda This way it will be visible as well when navigating using live documentation.